safe/impact_function/impact_function.py

```diff
@@ -800,6 +801,30 @@ def prepare(self):
                     )
                     return PREPARE_FAILED_BAD_INPUT, message
 
+            # We need to check if the hazard is OK to run on the exposure.
+            hazard_keywords = self.hazard.keywords
+            exposure_key = self.exposure.keywords['exposure']
+            if not active_thresholds_value_maps(hazard_keywords, exposure_key):
+                warning_heading = m.Heading(
+                    tr('Incompatible exposure/hazard'), **WARNING_STYLE)
+                warning_message = tr(
+                    'The hazard layer is not set up for this kind of exposure.'
+                    'In InaSAFE, you need to enable the set up the hazard '
+                    'classification for each kind of exposure you want to '
+                    'use.')
+                suggestion_heading = m.Heading(
+                    tr('Suggestion'), **SUGGESTION_STYLE)
+                suggestion = tr(
+                    'Please run the keyword wizard on the hazard layer to add '
```

Comment:
"Please run the select the hazard layer in the legend and then run the keyword wizard to define the needed keywords for this exposure."

It would be nice to also replace 'this exposure' with a string interpolator so that it reads e.g. "for **population** exposure".